.github/workflows/main.yml

```
@@ -15,6 +15,8 @@ jobs:
       run: python -m pip install -r requirements.txt
     - name: Lint
       run: pylama
+    - name: Test
+      run: pytest
```

When things fail, it's better if the logs are verbose:
```suggestion
      run: pytest -v
```